src/coreComponents/mesh/ElementRegionManager.cpp

```diff
@@ -542,5 +541,105 @@ ElementRegionManager::UnpackUpDownMaps( buffer_unit_type const * & buffer,
+                                                    string const fractureRegionName ) const
+{
+  int packedSize = 0;
+
+  SurfaceElementRegion const & embeddedSurfaceRegion =
+    this->getRegion< SurfaceElementRegion >( fractureRegionName );
+  EmbeddedSurfaceSubRegion const & embeddedSurfaceSubRegion =
+    embeddedSurfaceRegion.getSubRegion< EmbeddedSurfaceSubRegion >( 0 );
+
+  arrayView1d< globalIndex const > const embeddedSurfacesLocalToGlobal =
+    embeddedSurfaceSubRegion.localToGlobalMap();
+
+  packedSize += bufferOps::Pack< DOPACK >( buffer, numRegions() );
+
+  for( typename dataRepository::indexType kReg=0; kReg<numRegions(); ++kReg )
+  {
+    ElementRegionBase const & elemRegion = getRegion( kReg );
+    packedSize += bufferOps::Pack< DOPACK >( buffer, elemRegion.getName() );
+
+    packedSize += bufferOps::Pack< DOPACK >( buffer, elemRegion.numSubRegions() );
+    elemRegion.forElementSubRegionsIndex< CellElementSubRegion >(
+      [&]( localIndex const esr, CellElementSubRegion const & subRegion )
+    {
+      packedSize += bufferOps::Pack< DOPACK >( buffer, subRegion.getName() );
+
+      arrayView1d< localIndex const > const elemList = packList[kReg][esr];
+      if( DOPACK )
+      {
+        packedSize += subRegion.packFracturedElements( buffer, elemList, embeddedSurfacesLocalToGlobal );
+      }
+      else
+      {
+        packedSize += subRegion.packFracturedElementsSize( elemList, embeddedSurfacesLocalToGlobal );
+      }
+    } );
+  }
+
+  return packedSize;
+}
+
+int
+ElementRegionManager::unpackFracturedElements( buffer_unit_type const * & buffer,
+                                               ElementReferenceAccessor< localIndex_array > & packList,
+                                               string const fractureRegionName )
+{
+  int unpackedSize = 0;
+
+  SurfaceElementRegion & embeddedSurfaceRegion =
+    this->getRegion< SurfaceElementRegion >( fractureRegionName );
+  EmbeddedSurfaceSubRegion & embeddedSurfaceSubRegion =
+    embeddedSurfaceRegion.getSubRegion< EmbeddedSurfaceSubRegion >( 0 );
+
+  unordered_map< globalIndex, localIndex > embeddedSurfacesGlobalToLocal =
+    embeddedSurfaceSubRegion.globalToLocalMap();
+
+  localIndex numRegionsRead;
+  unpackedSize += bufferOps::Unpack( buffer, numRegionsRead );
+
+  for( localIndex kReg=0; kReg<numRegionsRead; ++kReg )
+  {
+    string regionName;
+    unpackedSize += bufferOps::Unpack( buffer, regionName );
+
+    ElementRegionBase & elemRegion = getRegion( regionName );
+
+    localIndex numSubRegionsRead;
+    unpackedSize += bufferOps::Unpack( buffer, numSubRegionsRead );
+    elemRegion.forElementSubRegionsIndex< CellElementSubRegion >(
+      [&]( localIndex const kSubReg, CellElementSubRegion & subRegion )
+    {
+      string subRegionName;
+      unpackedSize += bufferOps::Unpack( buffer, subRegionName );
+
+      /// THIS IS WRONG
```

Comment:
What should it be / why is it in here if it's WRONG?

Reply:
That comment `THIS IS WRONG??` is there in all the unpacking routines of the `ElementRegionManager` and I copy pasted that line. I am not sure why it would be wrong. I would actually say that they are all correct but I thought of leaving the comment in case someone decided to modify it later on.